modules/ROOT/pages/govcloud-data-control.adoc

```diff
@@ -0,0 +1,98 @@
+= GovCloud Data Control
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+MuleSoft Government Cloud (GovCloud) organizations can restrict the data entities exposed to their Salesforce Community Cloud ACM instance.
+
+Restrictions are defined on the Anypoint organization management plane and applied at the data bridge level.
+
+Before defining the entities and restrictions, get an authorization token from the Anypoint identity services API endpoint `/accounts/login` by sending the username and password of a user with the `Organization Administrator` permission.
+
+To define the restricted data entities and the operations allowed on each, use the token to make the API call to set a property in your Anypoint organization.
+
+Entities not listed in this configuration setting allow create, read, update, and delete operations.
+
+For entities listed in this configuration setting, all operations not present in the permissions array are blocked.
+
+Attempting a blocked operation returns an HTTP 403 error.
+
+== Example Request
```

Comment:
How is this a subtopic of the previous, in which requests are never mentioned?

Reply:
In this case API call is synonymous with request. Rewrote for clarity.